src/workflow/ArcanistDiffWorkflow.php

```diff
@@ -1861,6 +1861,29 @@ private function validateCommitMessage(
       }
     }
 
+    $config = $this->getConfigurationManager();
+    $mandatory_fields = $config->getConfigFromAnySource(
```

Comment:
Is this an `.arcconfig` source, or is this coming from somewhere else? Do individual repositories have the ability to turn this off?

If so, then :+1:.